components/kyma-environment-broker/common/runtime/model.go

```diff
@@ -154,7 +154,7 @@ type ListParameters struct {
 	// Expired parameter filters runtimes to show only expired ones.
 	Expired bool
 	// Events parameter fetches tracing events per instance
-	Events bool
+	Events string
```

Comment:
As I understand we are changing this in #2369. I think we can revert that change.